package.json

```diff
@@ -106,7 +106,7 @@
     "v-tooltip": "^2.0.3",
     "vue": "^2.6.14",
     "vue-click-outside-element": "^1.0.13",
-    "vue-color": "^2.8.1",
+    "vue-color": "https://github.com/apostrophecms/vue-color.git#allow-initial-null-vals",
```

Comment:
All dependencies, need to be published to npm before this is actually released. We can't have github dependencies in package.json of a shipping apostrophe release. That's because some of our clients don't allow use of git for module fetches inside their CI environment, as it's one more thing to keep secure and so on.

This concern can be cleared by publishing your version of vue-color as `vue-color-nullable` or similar, feel free to have a README that says it's hopefully only temporary and points to the upstream if you think best.

Reply:
Maybe `@apostrophecms/vue-color`? Definitely say it's temporary as possible.

Reply:
`@apostrophecms/vue-color` is better, yes. The README just needs to say it's intended as a temporary fork until the maintainers resume accepting fixes.